data/piper.1.in

```diff
@@ -0,0 +1,26 @@
+.TH PIPER 1 "@version_date@" "@version@" "User Commands"
+.SH NAME
+Piper \- GTK application to configure gaming devices
+.SH DESCRIPTION
+.SS "Usage:"
+.IP
+piper [OPTION]
+.SS "Help Options:"
+.TP
+\fB\-h\fR, \fB\-\-help\fR
+Show help options
+.TP
+\fB\-\-help\-all\fR
+Show all help options
+.TP
+\fB\-\-help\-gapplication\fR
+Show GApplication options
+.TP
+\fB\-\-help\-gtk\fR
+Show GTK+ Options
+.SS "Application Options:"
+.TP
+\fB\-\-display\fR=\fI\,DISPLAY\/\fR
+X display to use
```

Comment:
is it really worth it documenting those? they're pulled in and provided by GTK and, tbh, the use-case for `--display` has been dead for years.

Reply:
as an example: `evince` has a man page and doesn't document those. Because in most cases, these aren't useful to the user anyway.

Reply:
True, I will remove them. The reason why they are there is simply that I used help2man, which generated most of the man page from `piper --help`.